### PR TITLE
add missing publications

### PR DESCRIPTION
**Pull request by tamiko**

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -2682,4 +2682,18 @@ @mastersthesis{Vundla2018
...
+    author = {Matthias Maier and Andrei Nemilentsau and Tony Low and Mitchell Luskin},
+    doi = {10.1021/acsphotonics.7b01094},
+    journal = {ACS Photonics},
+    number = {2},
+    pages = {544--551},
+    title = {Ultracompact amplitude modulator by coupling hyperbolic polaritons over a graphene-covered gap},
+    url = {https://arxiv.org/abs/1709.06626},
+    volume = {5},
+    year = {2018}
+}
```
Comment (bangerth) [inline publications-2018.bib:2695]: Since it was published in a journal, I would suggest removing the URL to the preprint.

Comment (bangerth) [reply publications-2018.bib:2695]: same for the other publications

**Review by bangerth — CHANGES_REQUESTED** [review]

File: publications-2018.bib (hunk 1 above)
Comment (tamiko) [reply publications-2018.bib]: @bangerth Updated.

**Review by bangerth — APPROVED** [review]

**Pull request merged**